py/desiutil/test/test_dust.py

```diff
@@ -0,0 +1,55 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: utf-8 -*-
+"""
+desiutil.test.test_dust
+=======================
+
+Test desiutil.dust.
+"""
+from __future__ import absolute_import, division, unicode_literals
+# The line above will help with 2to3 support.
+import unittest
+import os
+import numpy as np
+from .. import dust
+
+
+class TestDust(unittest.TestCase):
```

Comment:
Assuming that `DUST_DIR` is defined, do the tests actually cover the entire module?  In other words, if we accept the module into desiutil, the total test coverage of desiutil should at least not decrease.

Reply:
I've added a unit test and coverage is now ~94%.